Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.9
+
+RUN apt-get update -y
+RUN apt-get upgrade -y
+RUN apt-get install postgresql-client -y
+
+COPY . /code
+RUN chmod +x /code/wait-for-it.sh
+WORKDIR /code
+
+RUN pip install poetry
+RUN poetry config virtualenvs.create false
+RUN poetry install --no-root
```

Comment:
We should probably also add a user and not run everything as root, I think this is an issue for later though

Reply:
There are some security concerns when running as root:

https://security.stackexchange.com/questions/176206/docker-runs-container-processes-as-root-should-i-be-worried
https://americanexpress.io/do-not-run-dockerized-applications-as-root/